<launch>
<arg name="robot" value="hal"/>

<group ns="$(arg robot)">
  <param name="language" value="es"/>
  
  <!-- tictactoe -->
  <remap from="drawer_in" to="drawer/in"/>
  <remap from="drawer_out" to="drawer/out"/>
  <node pkg="games_vision" type="launcher_hangman.exe"
        name="hangman"  output="screen"
        args="--display" />
  <node pkg="rostopic" type="rostopic" name="tictactoe_starter" output="screen"
        args="pub HANGMAN_START std_msgs/Int16 '{data: 0}' --latch  -1"/>

  <!-- TTS -->
  <remap from="tts" to="ETTS_SAY_PLAIN_TEXT" />
  <node pkg="games_vision" type="picotts.exe"
        name="picotts"  output="screen" />

  <!-- drawer -->
  <node pkg="games_vision" type="test_drawer.exe" name="drawer" output="screen"
        args=""/>
  <node pkg="rostopic" type="rostopic" name="drawer_starter" output="screen"
        args="pub DRAWER_START std_msgs/Int16 '{data: 0}' --latch  -1"/>
</group>
</launch>
